violetear_test.go

```diff
@@ -607,5 +607,44 @@ func TestVersioning(t *testing.T) {
 	req.Header.Set("Accept", "application/vnd.violetear.v2")
 	router.ServeHTTP(w, req)
 	expect(t, w.Body.String(), "* v2")
+}
+
+func TestGetParam(t *testing.T) {
+	testCases := []struct {
+		requestPath   string
+		expectedParam string
+	}{
+		{
+			requestPath:   "/tests/abc",
+			expectedParam: "abc",
+		},
+		{
+			requestPath:   "/other_test",
+			expectedParam: "",
+		},
+		// {
+		// todo: investigate, currently returns "tests" instead of ""
+		// 	requestPath:   "/tests/",
+		// 	expectedParam: "",
```

Comment:
Looks like that we have a problem here, since the param value returned equals the request path. I will tackle it separately, after we agree on implementation details.